src/test/java/org/eclipse/yasson/defaultmapping/generics/GenericsTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, 2023 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025 Oracle and/or its affiliates. All rights reserved.
```

Comment:
According to Eclipse Foundation, when omitting the second date, you must keep the remaining one unchanged.

Reply:
Ah, right. I had it backwards. I'll fix that to simply remove the second date and keep the first date.

Reply:
Well, that fails the copyright check. I'm not sure what the right approach is.

Reply:
This means, the copyright check is not compliant to the rules set up by the Eclipse Foundation, simply spoken. Things like this often happens if POMs were copied (or authored) from Oracle.

Reply:
@mkarg does the code change for the fix look ok? I have a user hitting this as a regression when moving from yasson2 to yasson3 , Thanks !

Reply:
It is up to the Yasson committers to decide.

Reply:
@m0mus According to Eclipse Foundation's rules, all projects MUST accept the short form having only the initial publication date (see https://www.eclipse.org/projects/handbook/#ip-copyright-headers). Apparently Yasson's POM.xml expects to find the latest date, which is wrong. What is your decision how to proceed?